### ISSUE TITLE
Authentication fails on Nuget.Server with Windows Auth (v3.4.2.830 & VS 2015)

### ISSUE THREAD

**Issue by sbochoa** [NuGet/Home#2529]
I just updated Nuget Package Manager to the last version and it still popups for windows credentials for my local nuget repository but it just keep asking credentials even when they are correct.

After 3 tries it just outputs "Response status code does not indicate success: 401 (Unauthorized)."

**Comment by emgarten** [comment]
@sbochoa what version of NuGet.Server are you using?

**Comment by yishaigalatzer** [comment]
@sbochoa can you please share a fiddler trace of before/after so we can try and repro?

We fixed a similar issue a little bit ago (not yet released), and I'd like to see if this is the same issue.

@alpaix can you please share a build here for verification?

**Comment by sbochoa** [comment]
@emgarten Using NuGet.Server.2.10.1

I just reinstalled Nuget Package Manager again and its working now kinda wierd if i reproduce it again ill share the fiddler trace. Thanks anyway

**Comment by yishaigalatzer** [comment]
Please consider updating the server, there are significant improvements in 2.10.2

**Comment by CheloXL** [comment]
I'm experiencing the same issue. Nuget.Server is 2.10.2 (although pointing a browser to my feeds it says "You are running NuGet.Server v2.10.0.0", not sure why).

Fiddler log says something like:
- Get on xxx/nuget, result 401
- Get on xxx/nuget with Authorization header, result 307 with a message that says "There is no operation listening for xxx/nuget, but there is an operation listening for xxx/nuget/, so you are being redirected there.
- Get on xxx/nuget/ WITHOUT authorization headers. Result is 401.

After the second 401, this starts to repeat until it fails (probably by number of retries).

**Comment by yishaigalatzer** [comment]
@alpaix can you please post the 3.4.3 fix here for verification that this was fixed for @CheloXL

**Comment by yishaigalatzer** [comment]
@CheloXL the v2.10.0.0 issue is going to get fixed in this commit - https://github.com/NuGet/NuGet.Server/commit/ae30d11f664017114a0874df74c4f818efc84e20 we will deploy to nuget.org shortly

**Comment by yishaigalatzer** [comment]
@CheloXL here are two private builds that should resolve the issue (hopefully) can you please try them out and let us know if it unblocks you?

### VSIX: Visual studio extension for Visual studio 2015

https://www.myget.org/F/nuget-vsix/vsix/NuGet.72C5D240-F742-48D4-A0F1-7016671E405B-3.4.3.835.vsix 

### NuGet.exe 3.4.3

https://www.myget.org/F/nugetbuild/api/v2/package/NuGet.CommandLine/3.4.3-rtm-844

**Comment by CheloXL** [comment]
@yishaigalatzer I'm getting "This extension is not installable on any currently installed products."

Error log:
15/04/2016 02:42:37 p.m. - Microsoft VSIX Installer
15/04/2016 02:42:37 p.m. - -------------------------------------------
15/04/2016 02:42:37 p.m. - Initializing Install...
15/04/2016 02:42:38 p.m. - Extension Details...
15/04/2016 02:42:38 p.m. -  Identifier         : NuGet.72C5D240-F742-48D4-A0F1-7016671E405B
15/04/2016 02:42:38 p.m. -  Name               : NuGet Package Manager for Visual Studio 15 Preview
15/04/2016 02:42:38 p.m. -  Author             : Microsoft Corporation
15/04/2016 02:42:38 p.m. -  Version            : 3.4.3.835
15/04/2016 02:42:38 p.m. -  Description        : A collection of tools to automate the process of downloading, installing, upgrading, configuring, and removing packages from a VS Project.
15/04/2016 02:42:38 p.m. -  Locale             : en-US
15/04/2016 02:42:38 p.m. -  MoreInfoURL        : 
15/04/2016 02:42:38 p.m. -  InstalledByMSI     : False
15/04/2016 02:42:38 p.m. -  SupportedFrameworkVersionRange : [0.0,2147483647.2147483647]
15/04/2016 02:42:38 p.m. - 
15/04/2016 02:42:38 p.m. -  SignatureState     : Unsigned
15/04/2016 02:42:38 p.m. -  Supported Products : 
15/04/2016 02:42:38 p.m. -      Microsoft.VisualStudio.Pro
15/04/2016 02:42:38 p.m. -          Version : [15.0,16.0)
15/04/2016 02:42:38 p.m. -      Microsoft.VisualStudio.Community
15/04/2016 02:42:38 p.m. -          Version : [15.0,16.0)
15/04/2016 02:42:38 p.m. - 
15/04/2016 02:42:38 p.m. -  References         : 
15/04/2016 02:42:38 p.m. - Signature Details...
15/04/2016 02:42:38 p.m. -  Extension is not signed.
15/04/2016 02:42:38 p.m. - 
15/04/2016 02:42:38 p.m. - Searching for applicable products...
15/04/2016 02:42:38 p.m. - Found installed product - Microsoft Visual Studio Enterprise 2015
15/04/2016 02:42:38 p.m. - Found installed product - Microsoft Visual Studio Professional 2015
15/04/2016 02:42:38 p.m. - Found installed product - Microsoft Visual Studio Community 2015
15/04/2016 02:42:38 p.m. - Found installed product - Microsoft Visual Studio 2015 Shell (Integrated)
15/04/2016 02:42:38 p.m. - Found installed product - Global Location
15/04/2016 02:42:38 p.m. - VSIXInstaller.NoApplicableSKUsException: This extension is not installable on any currently installed products.
   at VSIXInstaller.App.InitializeInstall(Boolean isRepairSupported)
   at VSIXInstaller.App.InitializeInstall()
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()

**Comment by yishaigalatzer** [comment]
Sorry my bad, this was the dev15 rather than visual studio 2015 version.

Please try this one:
https://www.myget.org/feed/nuget-vsix/package/vsix/NuGet.0d421874-a3b2-4f67-b53a-ecfce878063b/3.4.3.844

**Comment by CheloXL** [comment]
@yishaigalatzer Thanks, that worked. Also I can confirm that the above version works fine with my private nuget feed.

**Comment by alpaix** [comment]
Duplicate of https://github.com/NuGet/Home/issues/2521.
Fixed here: https://github.com/NuGet/NuGet.Client/pull/485.

**Comment by yishaigalatzer** [comment]
Thanks @CheloXL much appreciate the feedback and the time to report/verify!
